projects/addon-editor/components/editor-socket/editor-socket.component.ts

```diff
@@ -44,5 +47,32 @@ export class TuiEditorSocketComponent {
         @Optional()
         @Inject(TUI_SANITIZER)
         private readonly tuiSanitizer: Sanitizer | null,
+        @Inject(DOCUMENT)
+        private readonly document: Document,
+        @Optional()
+        @Inject(TuiTiptapEditorDirective)
+        private readonly editor: TuiTiptapEditorDirective | null,
     ) {}
+
+    /**
+     * @description:
+     * the main problem is that the external environment editor can use different base href="../"
+     * More information: https://rogerkeays.com/blog/using-base-href-with-anchors
+     */
+    @HostListener(`click`, [`$event`])
+    click(event: Event): void {
```

Comment:
Can't we just create content with discussed hack inline?

Reply:
I could not figure out how to do this through an existing Link extension

![telegram-cloud-photo-size-2-5381884254759274036-y](https://user-images.githubusercontent.com/12021443/197500627-c2a9daa4-70c2-4136-ae80-3844aef5e5c1.jpg)

Reply:
besides, it doesn't work like that if you try to anchor twice